aries-site/src/themes/aries.js

```diff
@@ -188,30 +168,195 @@ export const aries = {
+        medium: {
+          size: '69.2px',
+          height: '75.2px',
+          maxWidth: '1660.8px',
+        },
+        large: {
+          size: '120.4px',
+          height: '126.4px',
+          maxWidth: '2889.6px',
+        },
+        xlarge: {
+          size: '171.6px',
+          height: '177.6px',
+          maxWidth: '4118.4px',
+        },
+      },
+      '2': {
+        small: {
+          size: '30.8px',
+          height: '36.8px',
+          maxWidth: '739.2px',
+        },
+        medium: {
+          size: '43.6px',
+          height: '49.6px',
+          maxWidth: '1046.4px',
+        },
+        large: {
+          size: '69.2px',
+          height: '75.2px',
+          maxWidth: '1660.8px',
+        },
+        xlarge: {
+          size: '94.8px',
+          height: '100.8px',
+          maxWidth: '2275.2px',
+        },
+      },
+      '3': {
+        small: {
+          size: '24.4px',
+          height: '30.4px',
+          maxWidth: '585.6px',
+        },
+        medium: {
+          size: '30.8px',
+          height: '36.8px',
+          maxWidth: '739.2px',
+        },
+        large: {
+          size: '43.6px',
+          height: '49.6px',
+          maxWidth: '1046.4px',
+        },
+        xlarge: {
+          size: '56.4px',
+          height: '62.4px',
+          maxWidth: '1353.6px',
+        },
       },
-      2: {
-        small: fontSizing(1),
-        medium: fontSizing(3),
+      '4': {
+        small: {
+          size: '18px',
+          height: '24px',
+          maxWidth: '432px',
+        },
+        medium: {
+          size: '18px',
+          height: '24px',
+          maxWidth: '432px',
+        },
+        large: {
+          size: '18px',
+          height: '24px',
+          maxWidth: '432px',
+        },
+        xlarge: {
+          size: '18px',
+          height: '24px',
+          maxWidth: '432px',
+        },
       },
-      3: {
-        medium: fontSizing(1),
+      '5': {
+        small: {
+          size: '14.8px',
+          height: '20.8px',
+          maxWidth: '355.2px',
+        },
+        medium: {
+          size: '14.8px',
+          height: '20.8px',
+          maxWidth: '355.2px',
+        },
+        large: {
+          size: '14.8px',
+          height: '20.8px',
+          maxWidth: '355.2px',
+        },
+        xlarge: {
+          size: '14.8px',
+          height: '20.8px',
+          maxWidth: '355.2px',
+        },
+      },
+      '6': {
+        small: {
+          size: '11.6px',
+          height: '17.6px',
+          maxWidth: '278.4px',
+        },
+        medium: {
+          size: '11.6px',
+          height: '17.6px',
+          maxWidth: '278.4px',
+        },
+        large: {
+          size: '11.6px',
+          height: '17.6px',
+          maxWidth: '278.4px',
+        },
+        xlarge: {
+          size: '11.6px',
+          height: '17.6px',
+          maxWidth: '278.4px',
+        },
       },
     },
-    weight: 700,
```

Comment:
We will want to keep heading font-weight = 700 for the bold appearance.

Reply:
`weight` is specified on the base.js theme on 600, and I think we decided it to use it from there.

Reply:
I'm fine if the design side wants 700 for the HPE theme, leaving grommet base at 600.

Reply:
Thanks @halocline and @ericsoderberghp , theme got updated.